client/state/data-layer/wpcom-api-middleware.js

```diff
@@ -98,13 +98,14 @@ export const middleware = handlers => store => next => {
 		// this guarantees that we don't double-dispatch
 		const nextActions = new Set();
 		const safeNext = a => nextActions.add( a );
-
-		handlerChain.forEach( handler => handler( store, action, safeNext ) );
+		let shouldIgnore = false;
```

Comment:
how _dare_ someone introduce a `let` into my middleware!

🤣 

I'm just teasing with you, but hopefully someone who reviews this will get a laugh at that.

I'm only half-joking though… haha